tests/test_application/test_application.py

```diff
@@ -78,6 +80,26 @@ def test_application_contents_with_none_contents(flask_test_client):
     assert response.status_code == 400
 
 
+def test_application_upload_file_logic(app_context):
+    """
+    GIVEN: our service running with app_context fixture.
+    WHEN: we post expected data to Application class to map contents.
+    THEN: we check if expected contents are returned.
+    """
+
+    expected_json = expected_application_data
+    data = Notification.from_json(expected_json)
+    file_upload_func = Application.from_json(data)
+    questions = file_upload_func.questions
+
+    expected_contents_response = (
+        b"Funding service\n\n- about-your-org\n . Applicant name: Jack-Simon\n"
+        b" . Upload file: test-one_two.three/programmer.jpeg\n"
```

Comment:
Is this actually doing what we want? I would want the answer to only contain `programmer.jpeg`?

Reply:
yes, I added the funny file name 😆 and it only removes the key 

 "answer": "012ba4c7-e4971/test-one_two.three/programmer.jpeg"

Reply:
That's not what we're asserting, see `expected_contents_response`?

Reply:
I am testing the whole class of mapping contents instead of just file upload type